src/basic/log.c

```diff
@@ -1445,6 +1445,54 @@ static const char *const log_target_table[_LOG_TARGET_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(log_target, LogTarget);
 
+void log_task_chain(pid_t pid, const char *special_action_string) {
```

Comment:
I don't think this fits into `basic/log.[ch]`. The function serves a specific purpose, while the library is generic for setting up basic log functionality. Please find a better alternative or create a new one under `shared/`.

Reply:
I guess I will have to think of something under `shared/`.

Reply:
Moved under shared, as suggested.